### ISSUE TITLE
lmp needs an aktualizr-lite sota.toml for promoted builds

### ISSUE THREAD

**Issue by doanac** [foundriesio/meta-lmp#48]
I'm not sure the best way to handle this, but we should include a sota.toml example or maybe just default one that pulls in public ota updates. The contents should be:
~~~
[uptane]
polling_sec = 3600
repo_server = "https://api.foundries.io/ota/repo/lmp/api/v1/user_repo/"
key_source = "file"

[pacman]
type = "ostree"
ostree_server = "https://api.foundries.io/ota/treehub/lmp/api/v2/"
tags = promoted

[storage]
type = "sqlite"
path = "/var/sota/
~~~

**Comment by ricardosalveti** [comment]
We can probably have this settings available via its own package, and decide to install it or not via the CI job, what do you think?

**Comment by doanac** [comment]
sounds good

**Comment by ricardosalveti** [comment]
Fixed as part of #53
